libs-tests/package.json

```diff
@@ -1,28 +1,27 @@
 {
   "name": "lotus-eth-api-integration-tests",
   "version": "1.0.0-alpha.2",
+  "private": true,
   "description": "Integration tests for the Ethereum JSON-RPC API in Lotus",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "hardhat --network itest test"
```

Comment:
I have errors when I run 
```
mikers@mikers-B560-DS3H-AC-Y1:~/dev/rvagg/fevm-contract-tests/libs-tests$ npm run test

> lotus-eth-api-integration-tests@1.0.0-alpha.2 test
> hardhat --network itest test

Resetting Lotus node with block time: 200
Setting up new wallet for: 0x6be45e32a1e57b195be925570d5c68d8f29dc6e08859400b34e0e8b4abcc06b7
Sent 1000 to 0x7c5d5Aa6c0e37EBEc9b69594089869496dECcd71
Finished setup, Node RPC endpoint: http://127.0.0.1:34769


  ethers.js ERC20
    ✔ Should successfully deploy (681ms)
    ✔ Should access deployment transaction receipt
    ✔ Should set the right name
    ✔ Should set the right symbol
    ✔ Should set the right initial supply
    ✔ Should be able to transfer token (616ms)

  ethers.js SimpleCoin
    ✔ Should send deployment transaction (298ms)
    1) Should access transaction details before it has been mined
    2) Should access null transaction receipt before it has been mined
    ✔ Should successfully deploy
    ✔ Should access transaction details after it has been mined
    ✔ Should access transaction receipt after it has been mined
    ✔ Should find the transaction in block tx list
    ✔ Should get block tx count
    ✔ Should interact with the contract using eth_call
    ✔ Should get the contract byte code at the deployed address
    - Should get storage using eth_getStorageAt

  web3.js ERC20
    ✔ Should successfully deploy (1096ms)
    ✔ Should set the right name
    ✔ Should set the right symbol
    ✔ Should set the right initial supply
    ✔ Should be able to transfer token (1052ms)

  web3.js SimpleCoin
    ✔ Should send deployment transaction
    - Should access transaction details before it has been mined
    ✔ Should access null transaction receipt before it has been mined
    ✔ Should successfully deploy (1006ms)
    ✔ Should access transaction details after it has been mined
    ✔ Should access transaction receipt after it has been mined
    ✔ Should find the transaction in block tx list
    ✔ Should get block tx count
    ✔ Should interact with the contract using eth_call
    ✔ Should get the contract byte code at the deployed address
    - Should get storage using eth_getStorageAt


  28 passing (5s)
  3 pending
  2 failing

  1) ethers.js SimpleCoin
       Should access transaction details before it has been mined:
     AssertionError: expected '0xb1ddca070f520e0ff437b9e2a612cbad2dc…' to not exist
      at Object.should.not.exist (node_modules/chai/lib/chai/interface/should.js:208:38)
      at Object.testGetPendingTransactionByHash (test/util/testRpcResponses.js:13:14)
      at Context.<anonymous> (test/ethers.js/SimpleCoin.js:24:14)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)

  2) ethers.js SimpleCoin
       Should access null transaction receipt before it has been mined:
     AssertionError: expected { to: null, …(16) } to not exist
      at Object.should.not.exist (node_modules/chai/lib/chai/interface/should.js:208:38)
      at Object.testGetPendingTransactionReceipt (test/util/testRpcResponses.js:21:14)
      at Context.<anonymous> (test/ethers.js/SimpleCoin.js:31:14)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)



················································································································
|  Solidity and Network Configuration                                                                          │
···························|················|················|·················|································
|  Solidity: 0.8.23        ·  Optim: true   ·  Runs: 1000    ·  viaIR: false   ·   Block: 10,000,000,000 gas   │
···························|················|················|·················|································
|  Methods                                                                                                     │
···························|················|················|·················|················|···············
|  Contracts / Methods     ·  Min           ·  Max           ·  Avg            ·  # calls       ·  usd (avg)   │
···························|················|················|·················|················|···············
|  ERC20PresetFixedSupply  ·                                                                                   │
···························|················|················|·················|················|···············
|      transfer            ·     5,339,368  ·     5,503,031  ·      5,421,200  ·             2  ·           -  │
···························|················|················|·················|················|···············
|  Deployments                              ·                                  ·  % of limit    ·              │
···························|················|················|·················|················|···············
|  ERC20PresetFixedSupply  ·    30,594,317  ·    31,033,613  ·     30,740,749  ·         0.3 %  ·           -  │
···························|················|················|·················|················|···············
|  SimpleCoin              ·    14,059,428  ·    14,497,305  ·     14,190,791  ·         0.1 %  ·           -  │
···························|················|················|·················|················|···············
|  Key                                                                                                         │
················································································································
|  ◯  Execution gas for this method does not include intrinsic gas overhead                                    │
················································································································
|  △  Cost was non-zero but below the precision setting for the currency display (see options)                 │
················································································································
|  Toolchain:  hardhat                                                                                         │
················································································································
```

Reply:
I can't make this happen, it all passes for me! but I hope this shows up in CI so we can chalk it up as something to fix; for now I think this should be a "things to fix in lotus or in the libs" rather than here